### DISCUSSION TITLE
Ride selector with more details

### DISCUSSION THREAD

**Discussion by rondadams** [OpenRCT2/OpenRCT2#21759] [Feature requests]
It has always bothered me that the ride selector doesn’t provide more details like size, length, cost, etc in the list. Instead you have to click on each to see these details.
It would be nice if there was a selector with sort button like the attraction details dialog for in park (built) attractions. 

See mockup below:
The dropdown selector should have all (or most) of the ride attributes: Excitement, Intensity, Nausea, Max Speed, Average Speed, Drops, Inversions, Cost, Space Required, Ride Length. 

![RCTselector](https://github.com/OpenRCT2/OpenRCT2/assets/14861727/29dd9af5-9024-458c-bbb1-69899c7aa5d6)

**Comment by jcurbaniak** [comment]
This is a great idea.   I would like to be able to sort by cost when selecting a ride.  Maybe always show cost in a column.
